crate/guides/0.8.0/todomvc_update.md

```diff
@@ -267,9 +267,9 @@ And don't forget to check that everything works after each step as usual.
 
     - `input_element.get()` returns `Option<E>` where `E` is a specific DOM element reference like `web_sys::HtmlInputElement`. It returns `None` when the element doesn't exists in the DOM or has an incompatible type => all [ElRef](https://github.com/seed-rs/seed/blob/0a538f03d6aeb56b00d997c80a666e388279a727/src/virtual_dom/el_ref.rs) methods are safe to use.
 
-    - There are many `.expect(..)` calls because DOM operations are dangerous - any JS library or browser extension can modify DOM "under our hands", browsers have bugs and don't support all features in official specs, etc. So we want to get as much information as possible when the app panics because of such reasons. And descriptions inside `expect` calls help with readability.
+    - There are many `.expect(..)` calls because DOM operations are dangerous - any JS library or browser extension can modify the DOM "under our hands", browsers have bugs and don't support all features in official specs, etc. So we want to get as much information as possible when our app panics for one these reasons. Descriptions inside `expect` calls help with readability.
```

Comment:
```suggestion
    - There are many `.expect(..)` calls because DOM operations are dangerous - any JS library or browser extension can modify the DOM "under our hands", browsers have bugs and don't support all features in official specs, etc. So we want to get as much information as possible when our app panics for one of these reasons. Descriptions inside `expect` calls help with readability.
```